src/Components/App/App.js

```diff
@@ -35,6 +42,7 @@ function App() {
       <Routes>
         <Route path='/' element={<Movies movies={movies} showSingleMovie={showSingleMovie}/>} />
         <Route path='/movie/:id' element={<SingleMovie movie={singleMovie} goBack={goBack} />}/> 
+        <Route path='*' element={<ErrorHandling error={error}/>}></Route>
```

Comment:
Great job with this error handling. Nice way to catch any incorrect urls with the * symbol.